misc/test/examples_test.go

```diff
@@ -491,6 +512,14 @@ func TestAccAwsTsS3LambdaCopyZip(t *testing.T) {
 }
 
 func TestAccAwsTsServerlessRaw(t *testing.T) {
+	cmd := exec.Command("dotnet", "publish")
+	cmd.Path = path.Join(getCwd(t), "..", "..", "aws-ts-serverless-raw", "app")
+	cmd.Dir = path.Join(getCwd(t), "..", "..", "aws-ts-serverless-raw")
+	err := cmd.Run()
+	if err != nil {
+		fmt.Printf("err: %v", err)
+		return
+	}
```

Comment:
From conversation in Slack I think this needs to be deleted until https://github.com/pulumi/pulumi/pull/4559 is available and can be used instead.